server/routes/orderRoutes.js

```diff
@@ -2,12 +2,18 @@ import express from 'express';
 import ordersController from '../controllers/ordersController';
 import validators from '../middlewares/orderInputValidations';
 
+const {
```

Comment:
It's good your are deconstructing. But you can easily do this at line 2

Reply:
Thanks for the observation. I've made the adjustment and it's more tidy.